src/UI/StatusBar.re

```diff
@@ -240,15 +249,32 @@ let textItem = (~background, ~font, ~theme: Theme.t, ~text, ()) =>
   </item>;
 
 let notificationCount =
-    (~font, ~foreground as color, ~background, ~notifications, ()) => {
+    (
+      ~font,
+      ~foreground as color,
+      ~background,
+      ~notifications,
+      ~contextMenu,
+      ~onContextMenuUpdate,
+      (),
+    ) => {
   let text = notifications |> List.length |> string_of_int;
 
   let onClick = () =>
     GlobalContext.current().dispatch(
       Actions.StatusBar(NotificationCountClicked),
     );
+  let onRightClick = () =>
+    GlobalContext.current().dispatch(
+      Actions.StatusBar(NotificationsContextMenu),
+    );
 
-  <item onClick>
+  <item onClick onRightClick>
+    <Notifications.ContextMenu.Anchor
+      originY=`Top
+      model=contextMenu
+      onUpdate=onContextMenuUpdate
+    />
```

Comment:
This is just used as a reference point for where to render the menu. Nothing visible is rendered. It also has the added benefit of "live-updating" and moving the menu along with the anchor on resize and such (unlike the sneaks, for example).

Reply:
Cool!